### ISSUE TITLE
Crashing when opening map with only an automaton in the fleet

### ISSUE THREAD

**Issue by Bladewood** [endless-sky/endless-sky#2864]
That, or just a ship without any crew or that otherwise cannot be used as a flagship. I'm not sure.

[Doctor Hax~Test.txt](https://github.com/endless-sky/endless-sky/files/1185908/Doctor.Hax.Test.txt)

**Comment by tehhowch** [comment]
Blame your wanderlust
```
Thread 1 "endless-sky" received signal SIGSEGV, Segmentation fault.
0x00000000006462c0 in Ship::GetSystem (this=0x0) at source/Ship.cpp:1573
1573		return currentSystem;
(gdb) bt
#0  0x00000000006462c0 in Ship::GetSystem (this=0x0) at source/Ship.cpp:1573
#1  0x0000000000574c2e in MapPanel::DrawTravelPlan (this=0x3e50f90) at source/MapPanel.cpp:556
#2  0x0000000000571403 in MapPanel::Draw (this=0x3e50f90) at source/MapPanel.cpp:86
#3  0x000000000055e9d8 in MapDetailPanel::Draw (this=0x3e50f90) at source/MapDetailPanel.cpp:95
#4  0x0000000000690ac2 in UI::DrawAll (this=0x7fffffffcfe0) at source/UI.cpp:129
#5  0x00000000006972b4 in main (argc=1, argv=0x7fffffffdc28) at source/main.cpp:322
```

**Comment by endless-sky** [comment]
Fixed by merging #2867.
